apps/web/src/server/views/applications/representations/components/sort-button.njk

```diff
@@ -0,0 +1,28 @@
+{% macro sortButton(params) %}
+	{% set descValue = "-" + params.value %}
+	{% if params.queries.activeSort === params.value %}
+		<button
```

Comment:
the only difference between those buttons are some CSS classes - you can just add those classes on condition instead of repeating the whole BUTTON for each condition

Reply:
Good spot. 

Addressed. 

Can you see any more improvements on this ?

Reply:
added some comments on the CSS file